hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/utils/RationalizedSigMeta.java

```diff
@@ -144,7 +144,7 @@ public void revokeCryptoSigsFrom(final JKey key) {
     }
 
     public void replacePayerHollowKeyIfNeeded() {
-        if (!payerReqSig.hasHollowKey()) return;
+        if (payerReqSig == null || !payerReqSig.hasHollowKey()) return;
```

Comment:
I think we can log at `WARN` here; and then fix the NPE by moving [this line](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/logic/SigsAndPayerKeyScreen.java#L110) inside the following `if` block:

```
final var sigMeta = accessor.getSigMeta();

if (hasActivePayerSig(accessor)) {
    // Do not try to replace a hollow key if payer sig is not active
    sigMeta.replacePayerHollowKeyIfNeeded();
```

Reply:
Updated.

Reply:
Reverted the moving the `sigMeta.replacePayerHollowKeyIfNeeded()` inside the `if` because it makes all hollow account completion tests to fail.